### ISSUE TITLE
Documentserver 6.0.0 + Nextcloud 20 integration not working

### ISSUE THREAD

**Issue by smirkybg** [ONLYOFFICE/onlyoffice-nextcloud#387]
I'm using OO Documentserver 6.0.0 for integration with Nextcloud 20 and I cannot save the configuration page in Nextcloud, due to the following error:

`Error when trying to connect (Error occurred in the document service: Conversion error) (version 6)`

This happened after upgrading to 6.0.0 from 5.6.0. Reverting to 5.6.0 works and the issue is gone.
There is no significant output in the webserver logs, but nonetheless, this is what I observe upon clicking "Save":

```
192.168.1.1 - - [20/Oct/2020:11:07:55 +0300] "GET /healthcheck HTTP/1.1" 200 4 "-" "Nextcloud Server Crawler"
192.168.1.1 - - [20/Oct/2020:11:07:55 +0300] "POST /coauthoring/CommandService.ashx HTTP/1.1" 200 31 "-" "Nextcloud Server Crawler"
192.168.1.1 - - [20/Oct/2020:11:07:56 +0300] "POST /ConvertService.ashx HTTP/1.1" 200 80 "-" "Nextcloud Server Crawler"
```
I do, however, get an error from the converter service, which I obtained from journalctl:
```
Oct 20 11:26:41 archy converter[282237]: [2020-10-20T11:26:41.282] [ERROR] nodeJS - error spawnAsync(id=conv_check_1363532078_docx)
Oct 20 11:26:41 archy converter[282237]: Error: spawn /usr/share/webapps/onlyoffice/documentserver/server/FileConverter/bin/x2t ENOENT
Oct 20 11:26:41 archy converter[282237]:     at Process.ChildProcess._handle.onexit (internal/child_process.js:240:19)
Oct 20 11:26:41 archy converter[282237]:     at onErrorNT (internal/child_process.js:415:16)
Oct 20 11:26:41 archy converter[282237]:     at process._tickCallback (internal/process/next_tick.js:63:19)
Oct 20 11:26:41 archy converter[282237]: [2020-10-20T11:26:41.282] [ERROR] nodeJS - stdout (id=conv_check_1363532078_docx):
Oct 20 11:26:41 archy converter[282237]: [2020-10-20T11:26:41.282] [ERROR] nodeJS - stderr (id=conv_check_1363532078_docx):
Oct 20 11:26:41 archy converter[282237]: [2020-10-20T11:26:41.282] [ERROR] nodeJS - ExitCode (code=0;signal=null;error:-80;id=conv_check_1363532078_docx)
```

Enviroment details:
Kernel: 5.9.1
OS: Arch Linux
Nextcloud: 20.0.0
Nextcloud Onlyoffice Integration: 6.0.2
OnlyOffice DocumentServer: 6.0.0

Package details:
https://aur.archlinux.org/packages/onlyoffice-documentserver/#news

**Comment by ShockwaveNN** [comment]
@smirkybg Hi

Seems you using arch aur packages, which is supported via community, not via us, ONLYOFFICE team

I think it's better to report this problem in aur bug tracker

P.S. You can try to use Docker, our official way to run on arch

**Comment by smirkybg** [comment]
> @smirkybg Hi
> 
> Seems you using arch aur packages, which is supported via community, not via us, ONLYOFFICE team
> 
> I think it's better to report this problem in aur bug tracker
> 
> P.S. You can try to use Docker, our official way to run on arch

Hi, indeed I am. I did try the Docker version too little after writing this bug report. Unfortunately I failed to setup the integration, but I may try later again. I would, however, prefer to succeed with the AUR package, despite docker being somewhat easier for maintenance I suppose.

**Comment by ShockwaveNN** [comment]
> I would, however, prefer to succeed with the AUR package

Currently we got no plans to make our own AUR package, so you better contact current package community with your trouble. Thanks

**Comment by anothermouse** [comment]
I've just got a similar problem with Documentserver 6.0.0 when trying to use the build tools, and set up my own system pretty much from scratch

Problem was due to an incorrect path. Resolution process:
1. Identify where the bin/x2t binaries actually are using:    find /usr/share/webapps | grep x2t
2. If it isn't /usr/share/webapps/onlyoffice/documentserver/server/FileConverter/bin/x2t , then I copied the whole of the bin directory into the correct place. Note that I left the original bin directory where it was - Messy I know, but works!

If the above doesn't resolve the issue, then check permissions - there may be an issue with ownership, or access to the x2t binary file depending on how you carried out the install process.
